## ROS Topics Names
# Subscribers
grid_sub_topic: occupancy_grid
goal_sub_topic: goal

# Frame IDs
relative_frame_id: base_footprint

# Valid Moves & Associated Costs
moves: {
  list:   [west, east, north, south],
  costs:  [1.0, 1.0, 1.0, 1.0],
  
  north:      [ 1,  0],
  north_east: [ 1, -1],
  east:       [ 0, -1],
  south_east: [-1, -1],
  south:      [-1,  0],
  south_west: [-1,  1],
  west:       [ 0,  1],
  north_west: [ 1,  1]
}